### DISCUSSION TITLE
Simulation time is not stable

### DISCUSSION THREAD

**Discussion by onurg57** [cyberbotics/webots#5198] [Q&A]
When running simulation in normal speed, Simulation time running at 0.6x and suddenly it become 4-5x  and decrease again. While video capturing it works normal. Even while working with void controller.

**Comment by omichel** [comment]
This really depends on the simulation setup, including the controller program. It may be possible that at some steps the simulation runs computer intensive tasks, such as the rendering of a high resolution sensor. Running video capturing will slow down everything and may make the simulation run more steadily. Note: you could use the "run-realtime" mode to ensure the simulation speed won't exceed approximately 1x.
